pkg/debug/spec.sh

```diff
@@ -1,11 +1,73 @@
 #!/bin/sh
+# This script creates an initial hardware model file
+# The output is conservative when it comes to assignment groups in that
+# it does not form any assignment groups but merely checks whether there are
+# multiple functions on the same controller, and in that case the assignment
+# group is set to empty.
 
 if [ "$(uname -m)" = x86_64 ]; then
    ARCH=2
 else
    ARCH=4
 fi
 
+# log_to_dev returns the bus:controller without the function value
```

Comment:
Very very tiny nit -- it would be really great to indicate (at least in the comment section if not the name of the function) that this is all about PCI.

Also, your comment is misspelled `log_to_dev`

Reply:
Fixed